README.md

```diff
@@ -131,10 +131,13 @@ Then just call the `trap()` function in your code:
 ```php
```

Comment:
Please use `make lint-md` locally to push fixed README.md

<img width="657" alt="image" src="https://github.com/buggregator/trap/assets/94047334/7af1c7b8-4bf2-4ecf-bbc5-8effb398e820">